### DISCUSSION TITLE
Add searchable fields for recipes

### DISCUSSION THREAD

**Discussion by PATAPOsha** [Genymobile/feedback#80] [Genymotion SaaS] [answered]
Please, add some searchable fields/tags for recipe object, so it could be easier to manage big amount of recipes.
Currently, I have to write my searchable tags directly into the recipe's name. But the problem with name - is that it affects device user-agent. It could be critical for some apps. I'd like to leave user-agent as it is, e.g. "Pixel 3", but still be able to add some tags to recipe to filter/search them in the future.

**Comment by AlexandreDorys** [comment, marked as answer]
Hi,

Thank you for sharing your feedback. Improving the usability of recipes is on our radar, and your request aligns with our ongoing efforts to enhance the user experience. While I can't provide an exact timeline for when this feature will be implemented, your feedback strengthens our commitment to making these improvements.

Best regards,
Genymotion Team
